### PR TITLE
[CLEANUP] Clean up `blueprint.add-bower-package-to-project` and `blueprint.add-bower-packages-to-project` deprecations

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: lib/models/blueprint.js [hunk 1]
```diff
@@ -1143,119 +1141,6 @@ let Blueprint = CoreObject.extend({
...
-    may optionally have a `target` to specify a specific version, or a
-    `source` to specify a non-local name to be resolved.
-
-    @method addBowerPackagesToProject
-    @param {Array} packages
-    @param {Object} installOptions
-    @return {Promise}
-  */
-  addBowerPackagesToProject(packages, installOptions, _deprecationCondition = false) {
-    let task = this.taskFor('bower-install');
```
Comment (bertdeblock) [inline lib/models/blueprint.js:1220]: The `bower-install` task will be removed in a separate PR.

**Review by ef4 — APPROVED** [review]
